tests/test_integration/stencil_definitions.py

```diff
@@ -64,7 +64,7 @@ def afunc(b):
 @register
 def native_functions(field_a: Field3D, field_b: Field3D):
     with computation(PARALLEL), interval(...):
-        field_a = min(afunc(field_b), field_b)
+        field_a = max(min(afunc(field_b), field_b), 1)
```

Comment:
Is this enough? Isn't needed any kind of check of the results?

Reply:
Yes. When I put this test on the master branch, it actually fails with an error in the Exception that it raises :)
```
src/gt4py/analysis/passes.py:917: in visit_NativeFuncCall
    "Builtin function call with mixed data types", loc=node.loc
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = DataTypeSpecificationError('Builtin function call with mixed data types',)
data_type = 'Builtin function call with mixed data types', message = None

    def __init__(self, data_type, message=None, *, loc=None):
        if message is None:
            if loc is None:
                message = f"Invalid data type specification '{data_type}'"
            else:
>               message = f"Invalid data type specification '{data_type}' in '{loc.scope}' (line: {loc.line}, col: {loc.col})"
E               AttributeError: 'Location' object has no attribute 'col'
```

I fixed those now, and the test passes on this branch!